content/theme/Ephesus.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Ephesus"
+github: https://github.com/onepase/Ephesus
+demo: https://pardin.us/ephesus
```

Comment:
The demo must link to a demo website, not your own website. The demo website typically contains default content, not your own content.

Reply:
Demo contents updated.